hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/CryptoCreateHandlerTest.java

```diff
@@ -256,6 +279,39 @@ void preHandleWorksWhenInitialBalanceIsZero() throws PreCheckException {
         assertEquals(key, context.payerKey());
     }
 
+    @Test
+    @DisplayName("preHandle succeeds when has non zero evm alias")
+    void preHandleWorksWhenHasEvmAlias() throws PreCheckException {
+        final byte[] evmAddress = CommonUtils.unhex("6aeb3773ea468a814d954e6dec795bfee7d76e26");
+        txn = new CryptoCreateBuilder()
+                .withAlias(Bytes.wrap(evmAddress))
+                .withStakedAccountId(3)
+                .build();
+        final var context = new FakePreHandleContext(readableStore, txn);
+        subject.pureChecks(txn);
+        subject.preHandle(context);
+
+        assertEquals(txn, context.body());
```

Comment:
I'm seeing this in a few places. Preference is to favor assertj, especially for new lines of code